service/src/loader.rs

```diff
@@ -36,6 +40,11 @@ pub struct Loader {
 }
 
 impl Loader {
+    #[allow(dead_code)]
+    pub fn new(service: Box<dyn Service>, so: Library) -> Self {
+        Loader { service, so }
+    }
+
```

Comment:
```suggestion
```
This does not appear to be used :man_shrugging:

Reply:
Ah, my bad, a holdover from earlier testing :)